packages/components/data-table/src/data-table.tsx

```diff
@@ -93,7 +88,7 @@ export type TColumn<Row extends TRow = TRow> = {
    *
    * @@defaultValue@@: auto
    */
-  width?: number;
+  width?: number | string;
```

Comment:
Before the prop type was only `string`. Do we also need `number`? 🤔

Reply:
type number for width was need because some calculations was done with the width in `use-manual-column-resizing-reducer.ts`. It has been updated now though.